CONTRIBUTING.md

```diff
@@ -2,61 +2,159 @@
 
 :+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
 
-Following you will see some guidelines about how to contribute with
-JavaScript SDK.
+We welcome contributions from the community! Please take some time to become
+acquainted with the process before submitting a pull request.
 
-## Branch Management
+## Pull Requests
 
-We use Gitflow to manage our branches and that's ok when `develop` branch is
-ahead of `master`.
+When creating a pull request, first fork this repository and clone it to your
+local development environment. Then add this repository as the upstream.
 
-- [Gitflow](https://nvie.com/posts/a-successful-git-branching-model/) by @nvie
+```console
+git clone https://github.com/mygithuborg/sdk-javascript.git
+git remote add upstream https://github.com/cloudevents/sdk-javascript.git
```

Comment:
need a 'cd sdk-javascript' before this line

Reply:
good catch